nuxt.config.js

```diff
@@ -34,7 +34,11 @@ export default defineNuxtConfig({
     buildAssetsDir: '/nuxt/',
     head: {
       title: 'Centrapay Docs',
-      link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }],
+      link: [
+        { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' },
+        { rel: 'preconnect', href: 'https://fonts.gstatic.com', crossorigin: 'anonymous'},
+        { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap', crossorigin: 'anonymous'},
```

Comment:
I can see that in Tailwind example repos they have a `public/fonts/` directory with woff2 files stored that define the inter font - I'm not sure if this is a better way to do it 🤔

Reply:
@MeganSteenkamp Yes there's trade-off to self-hosting (no bug fixes, you might not have the right support for certain browsers) or downloading from Google APIs (performance impact). I've chosen the latter as the performance hit was ok - we can tune this later if we need to 👍